.github/workflows/octoguide.yaml

```diff
@@ -1,6 +1,5 @@
 jobs:
   octoguide:
-    if: ${{ !endsWith(github.actor, '[bot]') }}
```

Comment:
Err, is this not the opposite? Previously it was skipping bots. Do you _want_ it to check bot comments?

Reply:
Ah gotcha I see what you mean. Okie dokie!